src/palace/manager/celery/tasks/__init__.py

```diff
@@ -0,0 +1,22 @@
+import importlib
+from pathlib import Path
+
+
+def import_celery_tasks() -> None:
```

Comment:
This function had to get moved out `celery/app.py` so that it could be imported by the session worker fixture. That fixture cannot import `celery/app.py` because the fixture is initialized before we mock the services container.